components/ResetPassword.php

```diff
@@ -90,7 +89,7 @@ public function onRestorePassword()
             'code' => $code
         ];
 
-        Mail::send('winter.user::mail.restore', $data, function($message) use ($user) {
+        Mail::queue('winter.user::mail.restore', $data, function($message) use ($user) {
```

Comment:
What if no queue is set up?

Reply:
It should default to the sync driver: https://github.com/wintercms/winter/blob/develop/config/queue.php#L16

...unless it has been overridden for some reason. @LukeTowers @bennothommo should a separate notice be issued in the release notes?

Should we handle cases where a project uses no queues but still has the QUEUE_CONNECTION variable set to something other than sync?

Reply:
Yeah we can certainly add a note to the README that states that mail from this plugin will be queued.